src/jit-analyze/jit-analyze.cs

```diff
@@ -484,10 +606,11 @@ void DisplayMethodMetric(string headerText, int metricCount, dynamic list)
                 }
             }
 
-            DisplayMethodMetric("\nTop method regressions by size (bytes):", methodRegressionCount, sortedMethodRegressions);
-            DisplayMethodMetric("\nTop method improvements by size (bytes):", methodImprovementCount, sortedMethodImprovements);
-            DisplayMethodMetric("\nTop method regressions by size (percentage):", methodRegressionCount, sortedMethodRegressionsByPercentage);
-            DisplayMethodMetric("\nTop method improvements by size (percentage):", methodImprovementCount, sortedMethodImprovementsByPercentage);
+            DisplayMethodMetric("\nTop method regressions by codesize change  (bytes):", methodRegressionCount, sortedMethodRegressions);
+            DisplayMethodMetric("\nTop method improvements by codesize change (bytes):", methodImprovementCount, sortedMethodImprovements);
```

Comment:
I think you should write out "codesize" => "code size" and "perfscore" => "perf score" in the output

Reply:
I will write out "code size" and "PerfScore" in the output